pages/doc/integrations_tas_alerts.md

```diff
@@ -426,33 +409,31 @@ Whether an Auctioneer instance holds the expected Auctioneer lock (in Locket).
+This metric is complimentary to Active Locks, and it offers an Auctioneer-level version of the Locket metrics. Although it is emitted per Auctioneer instance, only 1 active lock is held by Auctioneer. Therefore, the expected value is 1. The metric can occasionally be 0 when the Auctioneer instances are performing a leader transition, but a prolonged value of 0 indicates an issue with Auctioneer.
 
 **Troubleshooting**
 1. Run `monit status` on the Diego Database VM to check for failing processes.
 2. If there are no failing processes, then review the logs for Auctioneer. 
-   * Recent logs for Auctioneer should show all but one of its instances are currently waiting on locks, and the active Auctioneer should show a record of when it last attempted to execute work. This attempt should correspond to app development activity, such as `cf push`.
-3. If you are unable to resolve the issue, pull logs from the Diego BBS and Auctioneer VMs, which includes the Locket service component logs, and contact VMware Tanzu Support.
+
+   Recent logs for Auctioneer should show all but one of its instances are currently waiting on locks, and the active Auctioneer should show a record of when it last attempted to execute work. This attempt should correspond to app development activity, such as `cf push`.
+3. If you can't resolve the issue, pull logs from the Diego BBS and Auctioneer VMs, which includes the Locket service component logs, and contact VMware Tanzu Support.
 
 ## TAS Locks Held by BBS
 
 Whether a BBS instance holds the expected BBS lock (in Locket). 
 * 1 means the active BBS server holds the lock
 * 0 means the lock was lost
 
-This metric is complimentary to Active Locks, and it offers a BBS-level version of the Locket metrics. 
-Although it is emitted per BBS instance, only 1 active lock is held by BBS. 
-Therefore, the expected value is 1. 
-The metric may occasionally be 0 when the BBS instances are performing a leader transition, but a prolonged value of 0 indicates an issue with BBS.
+This metric is complimentary to Active Locks, and it offers a BBS-level version of the Locket metrics. Although it is emitted per BBS instance, only 1 active lock is held by BBS. 
+Therefore, the expected value is 1. The metric can occasionally be 0 when the BBS instances are performing a leader transition, but a prolonged value of 0 indicates an issue with BBS.
 
 **Troubleshooting**
 1. Run `monit status` on the Diego database VM to check for failing processes.
 2. If there are no failing processes, then review the logs for BBS.
    * A healthy BBS shows obvious activity around starting or claiming LRPs.
-   * An unhealthy BBS leads to the Auctioneer showing minimal or no activity. The BBS sends work to the Auctioneer.
+   * An unhealthy BBS leads to the Auctioneer showing minimal or no activity. 
+     
+     The BBS sends work to the Auctioneer.
```

Comment:
Maybe something like:

Because the BBS sends work to the Auctioneer, an unhealthy BBS leads to the Auctioneer showing minimal or no activity.